client/components/plans/plan-icon/index.jsx

```diff
@@ -24,12 +24,35 @@ import {
 
 export default class PlanIcon extends Component {
 	render() {
+		let svg;
+		const iconPersonal = <svg  className="plan-icon__personal" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0" y="0" viewBox="0 0 124 125"><circle className="plan-icon__personal-0" cx="62" cy="62.1" r="62"/><path className="plan-icon__personal-1" d="M64 106.1l4-9.8v-3.5H62v14.6C62.9 107.4 63.6 106.9 64 106.1z"/><path className="plan-icon__personal-2" d="M56.2 92.8v3.5l4 9.8c0.3 0.8 1.1 1.3 1.9 1.3V92.8H56.2z"/><path className="plan-icon__personal-2" d="M62 32.9h17.1V2.5C73.7 0.9 68 0.1 62 0.1V32.9z"/><path className="plan-icon__personal-3" d="M62 32.9V0.1c0 0 0 0 0 0 -5.9 0-11.6 0.8-17 2.4v30.5H62z"/><polygon className="plan-icon__personal-4" points="62 32.9 45 32.9 48.3 48.7 53.3 92.8 62 92.8 "/><polygon className="plan-icon__personal-5" points="62 32.9 79.2 32.9 75.9 48.7 70.9 92.8 62 92.8 "/></svg>;
```

Comment:
Maybe move these out into helper methods like:
```jsx
getPersonalcon() {
    return (
          <svg //... />
    );
}
```

We can then move the switch into something like:

```jsx
getPlanIcon() {
    const { plan } = this.props;
    switch( plan ) {
        default: 
           return this.getFreeIcon();
        case: PERSONAL_PLAN:
        case: PLAN_JETPACK_PERSONAL:
        case: PLAN_JETPACK_PERSONAL_MONTHLY:
            return this.getPersonalIcon();
        //... etc
    }
}